index.html

```diff
@@ -35,21 +35,21 @@
           class="nav-search--input nav-search--box"
           placeholder="Search..."
         />
-        <button class="nav-search--button">
-          <i class="fas fa-search"></i>
-        </button>
+        <button class="nav-search--button">Search</button>
 
         <a href="#">Login</a>
       </div>
     </nav>
 
     <!-- CAROUSEL -->
-    <div class="slideshow-container">
-      <div class="mySlides">
-        <div class="numbertext">1 / 3</div>
-        <img src="https://placeimg.com/360/200/tech" />
-      </div>
-
+    <div class="carousel">
+      <figure class="carousel-container">
+        <img
+          src="https://placeimg.com/1000/600/tech"
+          alt="product-image"
+          class="carousel-container-image"
+        />
+      </figure>
       <a class="prev">&#10094;</a>
       <a class="next">&#10095;</a>
```

Comment:
Try adding pseudoelements with unicode content (`\u000`) instead of HTML entities ;)